level00/README.md

```diff
@@ -0,0 +1,30 @@
+# Level00
+
+This level is intened to serve as a terse introduction to 'property-based testing'. 
```

Comment:
Tighten prose: This level is a terse introduction to property-based testing.
If it fails at that, someone can file an issue.